.github/workflows/sonar-cloud-analysis.yml

```diff
@@ -22,6 +22,8 @@ jobs:
       matrix:
         include:
           - args: "-DOC_CLOUD_ENABLED=ON -DOC_COLLECTIONS_IF_CREATE_ENABLED=ON -DOC_PUSH_ENABLED=ON -DOC_RESOURCE_ACCESS_IN_RFOTM_ENABLED=ON -DPLGD_DEV_TIME_ENABLED=ON"
+          # logs enabled
+          - args: "-DOC_CLOUD_ENABLED=ON -DOC_COLLECTIONS_IF_CREATE_ENABLED=ON -DOC_PUSH_ENABLED=ON -DOC_RESOURCE_ACCESS_IN_RFOTM_ENABLED=ON -DPLGD_DEV_TIME_ENABLED=ON -DOC_DEBUG_ENABLED=ON"
```

Comment:
Logging code doesn't seem relevant for coverage statistics, in fact I would rather exclude it.

Reply:
For coverage. Otherwise it fails.